### ISSUE TITLE
(feature) - Allow chart version to diff to be specified

### ISSUE THREAD

**Issue by lachie83** [databus23/helm-diff#1]

**Comment by databus23** [comment]
I thought about this as well. Ideally it should be possible to compare arbitrary versions of a release.

I'm just not sure about the CLI UI for that without making it confusing. When comparing two *deployed* versions the command does not need the otherwise mandatory local directory.

**Comment by lachie83** [comment]
Currently doing something like this as a workaround

```shell
helm fetch --untar deis/workflow --version $(WORKFLOW_VERSION)
helm diff deis-workflow ./workflow \
-f ../../env/$(CLUSTER_ID)/workflow-values.yaml
```

**Comment by lachie83** [comment]
It also doesn't appear to trigger the hooks and display hook changes. Any ideas how hard that would be?

**Comment by lachie83** [comment]
@databus23 I've submitted a PR with the ability to specify the chart version. LMK what you think

**Comment by databus23** [comment]
I believe this has been fixed by a536dfc and is shipped since version 2.9.0+1 of this plugin.

**Comment by databus23** [comment]
I believe this has been fixed by a536dfc and is shipped since version 2.9.0+1 of this plugin.
